### PR TITLE
Update apps-selective-wipe.md

### PR DESCRIPTION
**Pull request by DanielEmerson**
Corrected Note - APP is required to enable app selective wipe for Android and iOS.

### CODE REVIEW

**Comment by PRMerger15** [comment]
@DanielEmerson : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/apps/apps-selective-wipe.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ ms.collection: M365-identity-device-management
 When a device is lost or stolen, or if the employee leaves your company, you want to make sure company app data is removed from the device. But you might not want to remove personal data on the device, especially if the device is an employee-owned device.
 
 >[!NOTE]
-> The iOS/iPadOS, Android, and Windows 10 platforms are the only platforms currently supported for wiping corporate data from Intune managed apps. Intune managed apps are applications that include the Intune APP SDK, and have at least one enabled and licensed user account in your organization. Deployment of Application Protection Policies are not required to enable app selective wipe.
+> The iOS/iPadOS, Android, and Windows 10 platforms are the only platforms currently supported for wiping corporate data from Intune managed apps. Intune managed apps are applications that include the Intune APP SDK, and have at least one enabled and licensed user account in your organization. Deployment of Application Protection Policies are  required to enable app selective wipe on Android and iOS.
```
Comment (cofitz1) [inline memdocs/intune/apps/apps-selective-wipe.md:39]: nit: extra space between `are  required`

**Review by cofitz1 — APPROVED** [review]

File: memdocs/intune/apps/apps-selective-wipe.md (hunk 1 above)
Comment (joakley-msft) [inline memdocs/intune/apps/apps-selective-wipe.md:39]: Agreement: should be "Deployment ... **is** required", not "Deployment ... are required"

**Comment by ktoliver** [comment]
Hi @DanielEmerson - This pull request was created in the public repo. The content has a private repository, and PMs usually work in the private repo. We can leave this PR here for review and merge, but would you make future content updates in the private repo? Thanks!

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by ktoliver** [comment]
I made the requested changes and will merge per author approval. Thanks, all.

**Pull request merged**